maquetado/scss/book-list.scss

```diff
@@ -0,0 +1,37 @@
+.book-list {
+  align-items: center;
+  display: flex;
+  flex-flow: row wrap;
+  max-height: 768px;
+  max-width: 1366px;
+  padding: 88px 215px 8px 215px;
+
+  .book-item-list {
+    align-items: flex-start;
+    box-shadow: 4px 4px 10px $black-russian;
+    display: flex;
+    flex-direction: column;
+    margin: 18px;
```

Comment:
I think that you need margin-right and margin-bottom only

Reply:
Why? Is not the same thing? I mean, margin-right: 18px and margin-left 18px is 36px and every single item are separated from your brother in the same distance on any direction. I can't see why is wrong but, I can change it anyway if you want to.

Reply:
We try to standardize to use margin-right to separate siblings when there are in row and margin-bottom when there are in column

Reply:
Done! Change that "all direction" margin for right and bottom.